dz/index.js

```diff
@@ -0,0 +1,48 @@
+const agregateUsersAndTodos = async(user) => {
+   const url = 'https://jsonplaceholder.typicode.com/users';
+   const todosUrl = 'https://jsonplaceholder.typicode.com/todos'
+   try {
+     const res = await fetch(url).then(Response => Response.json())
+     const todosRes = await fetch(todosUrl).then(Response => Response.json())
```

Comment:
1. don't use capital letter first in param's name. `response`  is a good variant 
2. this 2 calls must be done via `Promise.all`, because you don't need to wait for users to load todos